Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,55 @@
+7. There must be sufficient memory for the code.  It is unknown how much memory the code will occupy.  Operating code with libraries and compilation could be upwards of or slightly over 150 KB in Flash/Disk/Storage..  To account for the worst case, the board will need over 150 KB.
+8. A huge power-draw would mean that there needs to be a bigger power subsystem which could increase the cost to unacceptable levels.  The power requirements of the board need to be reasonably under or near 240 mW.  Here, we take a voltage supply of 12 V and current of 20 mA to get 240 mW.  12 V with 20 mA is from the power subsystem’s output to provide an adequate operation time or lifespan and  not damaged components.
+9. Some form of battery monitoring capabilities need to be included or applicable.  Telling the user how much longer the device may operate is a must.  This can be done using an additional component or algorithmic measurement.  The device will need to keep track of battery life to prevent data loss by saving and ceasing operations when reaching critical levels.  Doing this will ensure data protection.  
+10. The board itself cannot be too big.  “Too big” here is defined as having a volume greater than 50 cubic inches.  This size was set to prevent the device from proving too large.  50 cubic inches would be about the size of a small novella/book.  .  
+11. The manufacturer needs to be a reputable and safe choice.  Choosing a bad manufacturer could hurt the project.  A stable choice will lower the risk of receiving a malfunctioning board or running out of the specified board supply.  This device is planned to be open to public use and construction, so it will need to be trustworthy.
+12. The board will need to have coding language support for assembly, C, and or Python.  It is expected that the users will have some degree of coding experience and may potentially personalize or improve upon the code. These languages are chosen due to popularity and performance.  C, in particular, is predicted to be very desirable for this project.
+
+## BUILDABLE SCHEMATIC OF SUBSYSTEM
+Shown is the pseudo-code for the operating code.  The final form of the code may have parts added and additional functionalities included, but this is the general idea of how the code should flow.  Proceeding from each section is assuming that there are no errors detected indicating a “Success”.  If an error arises, a default handler will need to be called to assess the situation.  It should also be noted that the loop is intended to be theoretically infinite, however, there are explicit and implicit breaks at the battery running critically low (explicit) and the shutoff button being pushed (implicit).  “Timers” refer to either physical hardware or coded timers that go off to alert the system that the sensor(s) sampling is ready.
```

Comment:
Include the schematic here as well. Write a paragraph about the schematic, if only to say the schematic is shown in figure 27 (or whatever).